Libraries/Text/TextInput/Singleline/RCTUITextField.m

```diff
@@ -489,6 +489,17 @@ - (BOOL)becomeFirstResponder
   }
   return isFirstResponder;
 }
+
+- (BOOL)performKeyEquivalent:(NSEvent *)event
+{
+  // The currentEditor is NSText for historical reasons, but documented to be NSTextView.
+  NSTextView *currentEditor = (NSTextView *)self.currentEditor;
+  // The currentEditor is non-nil when focused and hasMarkedText means an IME is open.
+  if (currentEditor && !currentEditor.hasMarkedText && ![self.textInputDelegate textInputShouldHandleKeyEvent:event]) {
+    return YES; // Don't send currentEditor the keydown event.
+  }
+  return [super performKeyEquivalent:event];
+}
```

Comment:
Is the idea that you override this instead of `keyDown` here?

Reply:
OK If I override `keyDown` instead, it never gets hit... Not sure why but I guess this is by design then 😅